output/full.go

```diff
@@ -47,7 +47,11 @@ func submitFull(ctx context.Context, s snapshot.FullSnapshot, server *state.Serv
 	var err error
 	var data []byte
 
-	snapshotUUID := uuid.NewV4()
+	snapshotUUID, err := uuid.NewRandom()
```

Comment:
As long as we're making changes here, should we consider [v7](https://pkg.go.dev/github.com/google/uuid#NewV7)?

Reply:
I thought about that, but figured it might be best to keep that as a separate PR for clarity - though we could certainly ship it at the same time. @seanlinsley any concerns with switching to v7?

Reply:
No, having timestamp-prefixed UUIDs is generally good. We currently replace some random UUIDs at snapshot processing time with timestamp-prefixed versions because the collector doesn't.